pkg/pillar/dpcreconciler/linux.go

```diff
@@ -5,26 +5,27 @@ import (
 	"errors"
 	"fmt"
 	"net"
+	"os"
 	"strconv"
 	"strings"
 	"sync"
 	"syscall"
 	"time"
 
+	"github.com/Luzifer/go-openssl/v4"
```

Comment:
I don't see any changes to this package since 2020. Are we sure it is maintained?

Reply:
If we decide to use Go binary in the wwan container for decryption as opposed to OpenSSL (or no password encrypting for NIM->wwan exchange), then I will remove this dependency.